lib/modules/datasource/github-tags/cache.ts

```diff
@@ -26,9 +26,11 @@ query ($owner: String!, $name: String!, $cursor: String, $count: Int!) {
             target {
               ... on Commit {
                 hash: oid
```

Comment:
I guess we still need to return commit hash instead of tag hash (e.g., how it's reflected in GitHub UI)

Reply:
Is this done?